packit_service/worker/reporting.py

```diff
@@ -270,35 +268,32 @@ def _has_identical_comment(self, body: str, mode: DuplicateCheckMode) -> bool:
                     and body == comment.body
                 ):
                     return True
-
         return False
 
     def comment(
         self,
         body: str,
         duplicate_check: DuplicateCheckMode = DuplicateCheckMode.do_not_check,
+        to_commit: Optional[bool] = False,
```

Comment:
Is there any reason for `Optional[bool]` that I am missing? I think that just `bool` with a default value of `False` should be sufficient

Reply:
:facepalm: